gulpfile.js/tasks/browserify.js

```diff
@@ -38,6 +38,11 @@ var browserifyTask = function (callback, devMode) {
       bundleConfig = _.omit(bundleConfig, ['external', 'require'])
     }
 
+    if (!isDev) {
```

Comment:
I suspect it will cause us issues with source maps hence just adding it on PROD, potentially this conditional isn't need though.